go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -59,8 +61,9 @@ type destination struct {
 	// Based on this flag we can omit clusterSubnet exclusion from the related ACL.
 	// For dns-based rules, EgressDNS won't add ips from clusterSubnet to the address set.
 	clusterSubnetIntersection bool
-	nodeAddrs                 sets.Set[string]
-	nodeSelector              *metav1.LabelSelector
+	// nodeName: nodeIPs
+	nodeAddrs    map[string][]string
```

Comment:
why change to a map?

Reply:
so that you can delete nodeIP by knowing nodeName